src/nodeconductor_test/private_settings.py.example

```diff
@@ -1,12 +1,12 @@
 class AWSPrivateSettings(object):
-    access_key_id_name = 'some_key_id'
-    secret_access_key_name = 'some_access_key'
+    aws_access_key_id = 'my_aws_access_key_id'
+    aws_secret_access_key = 'my_aws_access_key'
```

Comment:
Reason for name changes:

Settings` objects in test cases are assembled using these classes from private settings, and also define some test case specific parameters. So, the final `Settings` object will look like

```
Settings:
  username: "foo"
  access_key: "bar"
```

We'll have a lot of pain once we decide to include more than one class from private settings, let's say, both `AWSPrivateSettings` and `FooPrivateSettings` that may also have access key for service Foo. In this case the property names in resulting `Settings` object will conflict. To avoid this, property names should be unique here.